pkg/sloop/processing/processing.go

```diff
@@ -55,6 +55,7 @@ func (r *Runner) Start() {
 			if err != nil {
 				r.processingFailed("cannot extract resource metadata", err)
 			}
+			glog.V(99).Infof("watchRec metadata: %v", resourceMetadata)
```

Comment:
I would suggest to add loglevel 99 in the constants file and use that. We already have it for other levels